pkg/util/paramtable/component_param.go

```diff
@@ -2149,9 +2168,9 @@ If this parameter is set false, Milvus simply searches the growing segments with
 	p.UpdateCollectionLoadStatusInterval = ParamItem{
 		Key:          "queryCoord.updateCollectionLoadStatusInterval",
 		Version:      "2.4.7",
-		DefaultValue: "5",
+		DefaultValue: "300",
 		PanicIfEmpty: true,
-		Doc:          "5m, max interval of updating collection loaded status for check health",
+		Doc:          "300s, max interval of updating collection loaded status for check health",
```

Comment:
it's dangerous for users which already set this value to 5 as 5minutes, recomand to set up new param item if necessary

Reply:
The parameter likely wasn't used in the previous version, and even if it was, setting it to 5 seconds is irrelevant as it depends on the CheckHealth request from the client side.

Reply:
The previous minute unit for testing is impractical, so it will be changed to seconds.